lib/init.js

```diff
@@ -39,7 +44,10 @@ module.exports = function(grunt) {
     if (grunt.config('config.siteUrls') === undefined)
       grunt.config('config.siteUrls', {});
 
-    grunt.config('config.domain', module.domain());
+    grunt.config('config.domain', module.magic('domain') || require('os').hostname());
+    grunt.config('config.alias', module.magic('alias', 'SITE_ALIAS') || '@' + grunt.config('config.domain'));
+    grunt.config('config.project.profile', module.magic('profile', 'INSTALL_PROFILE') || 'standard');
```

Comment:
Initially this was "config.profile", but since grunt install added config.project.profile I switched this. However, the "magic" picks up on gdt.profile, so maybe one or the other needs a further change.